sonar-project.properties

```diff
@@ -5,9 +5,7 @@ sonar.projectVersion=0.0.1
 sonar.sources=app
 sonar.tests=test
 sonar.language=js
-sonar.exclusions=test/**, app/assets/**
+sonar.exclusions=test/**, app/assets/**, app/steps/ui/tasklist/includes/**, app/steps/ui/executors/roles/template.html
```

Comment:
As my other branch will probably get merged first, you'll have to update these.

Reply:
done